internal/operands/metrics/resources.go

```diff
@@ -82,7 +83,7 @@ func newServiceMonitorCR(namespace string) *promv1.ServiceMonitor {
 					Scheme: "https",
 					TLSConfig: &promv1.TLSConfig{
 						SafeTLSConfig: promv1.SafeTLSConfig{
-							InsecureSkipVerify: true,
```

Comment:
It's ok to skip TLS?

Reply:
I'm not sure, this is old code. I will open an issue to investigate.

Reply:
https://github.com/kubevirt/ssp-operator/issues/1009